tasks/screenshot-components.mjs

```diff
@@ -0,0 +1,81 @@
+import { join } from 'path'
+import { launch } from 'puppeteer'
+import percySnapshot from '@percy/puppeteer'
+import { isPercyEnabled } from '@percy/sdk-utils'
+
+import { getDirectories, getListing } from '../lib/file-helper.js'
+import { goToComponent } from '../lib/puppeteer-helpers.js'
+
+import configPaths from '../config/paths.js'
+import configPuppeteer from '../puppeteer.config.js'
+
+/**
+ * Send all component screenshots to Percy
+ * for visual regression testing
+ *
+ * @returns {Promise<void>}
+ */
+export async function screenshotComponents () {
+  const browser = await launch(configPuppeteer.launch)
+  const componentNames = await getDirectories(configPaths.components)
+
+  // Screenshot each component
+  const input = [...componentNames]
+
+  // Screenshot 4x components concurrently
```

Comment:
How did you arrive at 4x? Might be useful to capture in the commit message in case we want to revisit in the future.

Reply:
Tried 5x initially but 4x was faster

Good point though, worth trying 2x and 3x as well (you never know!)

Reply:
If you click into the **Send screenshots to Percy** job to see the timings:

* 1x component in sequence [govuk-frontend/actions/runs/5854565851](https://github.com/alphagov/govuk-frontend/actions/runs/5854565851) – ⏱️ **36 seconds**
* 2x components concurrently [govuk-frontend/actions/runs/3496480973](https://github.com/alphagov/govuk-frontend/actions/runs/3496480973) – ⏱️ **31 seconds**
* 3x components concurrently [govuk-frontend/actions/runs/3496481529](https://github.com/alphagov/govuk-frontend/actions/runs/3496481529) – ⏱️ **28 seconds**
* 4x components concurrently [govuk-frontend/actions/runs/3496493071](https://github.com/alphagov/govuk-frontend/actions/runs/3496493071) – ⏱️ **22 seconds**
* 5x components concurrently [govuk-frontend/actions/runs/5854480472](https://github.com/alphagov/govuk-frontend/actions/runs/5854480472) – ⏱️ **22 seconds**

The total action time varies due to [GitHub cache retrieval](https://github.com/alphagov/govuk-frontend/actions/caches) API times being wobbly

I'll keep it at 4x but add a comment